src/Service/SiteService.php

```diff
@@ -303,4 +307,9 @@ public function getSiteType()
     {
         return $this->isDVType() ? 'dv' : 'hpo';
     }
+
+    public function displayCaborConsent(): bool
+    {
+        return $this->session->get('siteState') === self::CABOR_STATE ? false : true;
```

Comment:
Isn't this backwards? CABoR consent should be displayed for CA sites, and only for CA sites.

Reply:
Yes, you are right. I guess I got tricked by the session variable not being set while testing.